Software-Engineering/L4-Lead-Software-Engineer.md

```diff
@@ -45,15 +45,16 @@
 
 ## 🧭 Culture and Leadership
 
-- I encompass all of the qualities listed in [Leadership](https://github.com/OctopusDeploy/People/blob/main/Leadership.md)
-- I avoid becoming a bottleneck, by setting up other engineers on my team for success for longer periods of time before needing to check back in
-- I make sure difficult work that needs to be done gets done, like hard debugging, incident response, and reverse-engineering tasks
+- I encompass all of the qualities listed in [Leadership](https://github.com/OctopusDeploy/People/blob/main/Leadership.md).
+- I avoid becoming a bottleneck, by setting up other engineers on my team for success for longer periods of time before needing to check back in.
+- I make sure difficult work that needs to be done gets done, like hard debugging, incident response, and reverse-engineering tasks.
 - I understand our organisation structure and how we schedule work, and help my team find their highest value contributions in this environment.
 - I am comfortable with transparently assessing risk, making recommendations, escalating appropriately, and dealing with the consequences along the way.
 - I recognise problems and proactively engage with other people to prevent or resolve them quickly.
 - I am comfortable coordinating projects where success requires the input of multiple teams and diverse technical roles.
 - I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
 - I proactively improve and implement processes that help uplift the way my team works together.
+- I collaborate effectively with team members from the Product and Design functions. I help to guide these team members towards the outcomes I need of them, and I ensure their needs are met in the work I do.
```

Comment:
Could this be more generic so it's not just applicable to Product + Design but any other functions. That would mean if teams were working with web developers in marketing or such it's still applicable.

Perhaps even:
> I collaborate effectively with team members from **other functions (e.g. Product and Design)**

Reply:
Nice call out, thanks for helping extend the thinking beyond Product and Design 👍